keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -293,7 +293,7 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
             }
             val invalidPayload = mapper
               .readValue<Map<String, Any?>>(payload)
-              .let { it - "serviceAccount" }
+              .let { it - "application" }
```

Comment:
A couple of tests that checked required field handling needed to change to look at a new field